react/ConditionLayoutProduct.tsx

```diff
@@ -32,6 +33,7 @@ type SubjectArgs = {
   selectedItemId: { id: string }
   specificationProperties: { name: string; value?: string }
   areAllVariationsSelected: undefined
+  isProductAvailable: boolean
```

Comment:
This is closer! We should have no arguments here, so `undefined`
```suggestion
  isProductAvailable: undefined
```